src/components/NewMovie/NewMovie.js

```diff
@@ -1,13 +1,109 @@
 import React, { Component } from 'react';
+import './NewMovie.scss';
+import PropTypes from 'prop-types';
 
 export class NewMovie extends Component {
-  state = {};
+  state = {
+    title: '',
+    description: '',
+    imgUrl: '',
+    imdbUrl: '',
+    imdbId: '',
+  };
+
+  handleSubmit = () => {
+    const { addMovie } = this.props;
+
+    addMovie(this.state);
+
+    this.setState({
+      title: '',
+      description: '',
+      imgUrl: '',
+      imdbUrl: '',
+      imdbId: '',
+    });
+  }
 
   render() {
     return (
-      <form>
-        Put the form here
+      <form
+        className="form"
+        onSubmit={(event) => {
+          event.preventDefault();
+          this.handleSubmit();
+        }}
+      >
+        <input
+          className="input"
+          type="text"
+          placeholder="Title"
+          value={this.state.title}
+          onChange={(event) => {
+            this.setState({
+              title: event.target.value,
```

Comment:
```suggestion
            this.setState({
              title: event.target.value,
```
Вынеси в метод 
handleChange = (event) => {
    const { name, value } = event.target;

    this.setState({
      [name]: value,
    });
  }

Reply:
Можно будет это во все вставить , просто Name сделай такой же как и поле в стейте

Reply:
а так все круто

Reply:
done)